### ISSUE TITLE
snake case warning triggering on... more ununsual forms.

### ISSUE THREAD

**Issue by ityonemo** [rrrene/credo#696]
### Environment

* Credo version (`mix credo -v`):
1.1.3
* Erlang/Elixir version (`elixir -v`):
1.9.1
* Operating system:
linux

### What were you trying to do?

```
  defmacro @{_, _, _} do:
     ...code...
```

see https://github.com/ityonemo/state_server/blob/master/lib/state_server/at_state_graph_macro.ex

### Expected outcome
no complaint

### Actual outcome
```
Module attribute names should be written in snake_case.
```

**Comment by ityonemo** [comment]
will submit PR soon.

**Comment by ityonemo** [comment]
https://github.com/rrrene/credo/pull/697

**Comment by rrrene** [comment]
Thx you very much for this! 👏 

Your changes are released as part of `v1.1.4`!
